assets/tests/components/map.test.tsx

```diff
@@ -106,14 +51,16 @@ afterAll(() => {
   global.scrollTo = originalScrollTo
 })
 
-describe("map", () => {
+describe("<Map />", () => {
```

Comment:
I'm considering reverting this and the describe in vehiclePropertiesCard, given it doesn't match style of the rest of the application tests.

Reply:
I'm reconsidering this given that it's helpful seeing when something being tested is a component, especially helpful for nested component testing.